staging/kudo/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: "1.0"
+description: "Kubernetes Universal Declarative Operator"
+name: kudo
+version: 0.7.4
```

Comment:
It seels like this and the `appVersion` should be swapped, no? `version` is the Chart version, and since this is the first version `0.1.0` or something like that seems reasonable

Reply:
good point